chainconfig/chainconfig.go

```
@@ -65,6 +75,22 @@ func (cc ChainConfig) Validate() error {
 		return errors.New("DAGasPriceDeviationPPB not set or negative")
 	}
 
+	if cc.ExecNoDeviationThresholdUSDWei.Int == nil {
+		return errors.New("ExecNoDeviationThresholdUSDWei not set. Default value is 10e9")
+	}
+
+	if cc.ExecNoDeviationThresholdUSDWei.Int.Cmp(big.NewInt(0)) < 0 {
+		return errors.New("ExecNoDeviationThresholdUSDWei not set or negative. Default value is 10e9")
+	}
+
+	if cc.DataAvNoDeviationThresholdUSDWei.Int == nil {
+		return errors.New("DataAvNoDeviationThresholdUSDWei not set. Default value is 20e9")
+	}
+
+	if cc.DataAvNoDeviationThresholdUSDWei.Int.Cmp(big.NewInt(0)) < 0 {
+		return errors.New("DataAvNoDeviationThresholdUSDWei not set or negative. Default value is 20e9")
+	}
+
```

Is it worth setting the defaults here instead of returning a failure?